tools/src/post-mortem/Main.hs

```diff
@@ -36,12 +39,37 @@ main = do
             err $ "wait window: " ++ show waitWindow
             err ""
             for_ logDirs $ processLogDirThroughput txWindow waitWindow
+        TestPostProcess logDir -> do
+            postProcessLogs logDir
+            err "done"
```

Comment:
I find a bit strange signalling the fact you are "done" by using `err`. What does `err` do? Cannot we simply print using `putText` or similar? Ah, I think `err` might be simply logging to `stderr`, is that right? In which case I think it's fine 😉

Reply:
yeah, all of the existing code uses err, which is just an alias for print to stderr